udata/templates/organization/display_base.html

```diff
@@ -1,6 +1,7 @@
 {% extends theme("layouts/1-column.html") %}
 {% from theme('macros/follow.html') import follow_btn with context %}
 
+{% set bundle = 'organization' %}
```

Comment:
Isn't the `bundle` declaration redundant with the `body_class` one? Can we merge these statements into one?